ecommerce/api.py

```diff
@@ -2,18 +2,101 @@
 Functions for ecommerce
 """
 from base64 import b64encode
+from datetime import datetime
+from decimal import Decimal
 import hashlib
 import hmac
 import logging
+import uuid
 
 from django.conf import settings
+from django.db import transaction
+from django.db.models.aggregates import Sum
+import pytz
+from rest_framework.exceptions import ValidationError
+
+from backends.utils import get_social_username
+from ecommerce.models import (
+    Line,
+    Order,
+)
+from klasses.models import Klass
 
 
 ISO_8601_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 log = logging.getLogger(__name__)
 _REFERENCE_NUMBER_PREFIX = 'BOOTCAMP-'
 
 
+def get_total_paid(user, klass_id):
+    """
+    Get total paid for a klass for a user
+    Args:
+        user (User):
+            The purchaser of the klass
+        klass_id (int):
+            A klass id, a class within a bootcamp
+
+    Returns:
+        Decimal: The total amount paid by a user for a klass
+    """
+    return Line.objects.filter(
+        order__status=Order.FULFILLED,
+        order__user=user,
+        klass_id=klass_id,
+    ).aggregate(price=Sum('price'))['price'] or Decimal(0)
```

Comment:
yeah... but a classmethod would be the same, I guess... feel free to leave as is anyway